src/platform/shared/common.c

```diff
@@ -1,4 +1,3 @@
-#include <malloc.h>
```

Comment:
Why remove it?

Reply:
Lots of info here https://stackoverflow.com/a/12973403

You can get malloc from stdlib